addons/udes_priorities/models/stock_picking.py

```diff
@@ -0,0 +1,124 @@
+from odoo import fields, models, api, _
+from collections import defaultdict, OrderedDict
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    @api.model
+    def _default_priority(self):
+        Priorities = self.env["udes_priorities.priority"]
+        priorities = Priorities.search(self._priority_domain(), limit=1)
+        return priorities.reference if priorities else None
+
+    priority = fields.Selection(selection="get_priorities_for_selection", default=_default_priority)
```

Comment:
I would consider renaming `get_priorities_for_selection` or `get_possible_priorities` from stock.move for consistency.